website/content/labs/lab9.md

```diff
@@ -30,7 +30,7 @@ You should already have the "upstream" remote set up in your repository. If you
 
 to get the files for this lab. If you don't have the "upstream" remote set up, follow task 0 from [Lab 2]({{< relref "lab2.md" >}}).
 
-The `lab9` directory contains a `libgeometry` directory with an updated version of `libgeometry` that has been documented in a way that allows Doxygen to automatically produce documentation based on it.
+The `labs/lab9` directory contains a `libgeometry` directory with an updated version of `libgeometry` that has been documented in a way that allows Doxygen to automatically produce documentation based on it. Please note that this version is distinct from the one in the root of your repository. You should do your work for this week's lab in the `labs/lab9/libgeometry/` directory, not in the top-level `libgeometry` directory.
 
```

Comment:
`doxygen` wasn't installed on the VM - they just need to run `sudo apt install doxygen`